frontend/components/Filter/FilterDialog.tsx

```diff
@@ -2,331 +2,413 @@
 
 import { Dialog, Transition } from "@headlessui/react";
 import {
-  AdjustmentsHorizontalIcon,
-  XMarkIcon,
+    AdjustmentsHorizontalIcon,
+    XMarkIcon,
 } from "@heroicons/react/24/outline";
-import { Checkbox, Slider, Input } from "@material-tailwind/react";
-import { Fragment, useState } from "react";
+import { Slider, Input } from "@material-tailwind/react";
+import { Fragment } from "react";
 import Datepicker from "react-tailwindcss-datepicker";
 import ListBox from "../../components/ListBox";
 import { EventData } from "@/interfaces/EventTypes";
-import {
-  filterEventsByDate,
-  filterEventsByMaxProximity,
-  filterEventsByPrice,
-} from "@/services/filterService";
-import { Timestamp } from "firebase/firestore";
-import {
-  SYDNEY_LAT,
-  SYDNEY_LNG,
-  getLocationCoordinates,
-} from "@/services/locationUtils";
-import { set } from "firebase/database";
+import { NO_SPORT_CHOSEN_STRING } from "@/services/filterService";
+
+export enum SortByCategory {
+    HOT,
+    TOP_RATED,
+    PRICE_ASCENDING,
+    PRICE_DESCENDING,
+    DATE_ASCENDING,
```

Comment:
Change sort by filter name to newest first/oldest first rather than date ascending and date descending. This can be fixed in another ticket as might be a lot of changes required

Reply:
If so, make another ticket for it